modules/pbm/bitmap_loader_pbm.cpp

```diff
@@ -95,7 +95,7 @@ static bool _get_token(FileAccessRef &f, uint8_t &saved, PoolVector<uint8_t> &r_
 				resized = true;
 			}
 			if (resized) {
-				w = PoolVector<uint8_t>::Write();
+				PoolVector<uint8_t>::Write();
 				r_token.resize(token_max);
 				w = r_token.write();
```

Comment:
This is a bug. `PoolVector<uint8_t>::Write()` has an [`operator=` method](https://github.com/godotengine/godot/blob/master/core/dvector.h#L329).

So here this actually performs side effects and is not just a weird mistake.